hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/TestLargeClusterBalancingConditionalReplicaDistribution.java

```diff
@@ -96,11 +96,10 @@ private static TableName getTableName(int i) {
   @Test
   public void testReplicaDistribution() {
     Configuration conf = new Configuration(true);
-    conf.setBoolean(BalancerConditionals.DISTRIBUTE_REPLICAS_KEY, true);
-    conf.setBoolean(DistributeReplicasConditional.TEST_MODE_ENABLED_KEY, true);
+    DistributeReplicasTestConditional.enableConditionalReplicaDistributionForTest(conf);
     conf.setBoolean(ReplicaKeyCache.CACHE_REPLICA_KEYS_KEY, true);
     conf.setInt(ReplicaKeyCache.REPLICA_KEY_CACHE_SIZE_KEY, Integer.MAX_VALUE);
-    conf.setLong("hbase.master.balancer.stochastic.maxRunningTime", 30_000);
+    conf.setLong("hbase.master.balancer.stochastic.maxRunningTime", 120_000);
```

Comment:
According to the annotation, a LargeTest is permitted to run up to 10 minutes (I think that's the whole class). You have some room to work with if it means the test will run reliably on our CI environment.

Reply:
Because the balancer will iterate, it doesn't really matter what this is set to (as long it's not far too short or indefinitely long I suppose). So I'm going to change this back, I was just playin' around and committed this by accident